src/main/java/ooo/oshi/foreign/windows/Kernel32Library.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributions to this file must be licensed under the Apache 2.0 license or a compatible open source license.
+ */
+package ooo.oshi.foreign.windows;
+
+import java.lang.foreign.*;
+import java.lang.invoke.MethodHandle;
+
+import static java.lang.foreign.ValueLayout.JAVA_INT;
+
+public class Kernel32Library {
+
+    private static final SymbolLookup Kernel32;
+    private static final Linker linker;
+
+    static {
+        linker = Linker.nativeLinker();
```

Comment:
not really necessary storing this since it's a singleton...

Reply:
Maybe a global constant that we an import statically everywhere?

As usual with most code things, started from an example, grew from there, and eventually figured something needed refactoring.... rinse and repeat!